INFO 2024-11-07 16:30:20,096 autoreload Watching for file changes with StatReloader
WARNING 2024-11-07 16:31:45,771 log Not Found: /favicon.ico
ERROR 2024-11-07 16:31:49,930 mailchimp Mailchimp API error: {"type":"https://mailchimp.com/developer/marketing/docs/errors/","title":"Resource Not Found","status":404,"detail":"The requested resource could not be found.","instance":"860d461b-6796-e325-8aaa-87f29c9d8b63"}
INFO 2024-11-07 16:36:02,984 autoreload /workspace/cosmic-love-shop/users/mailchimp.py changed, reloading.
INFO 2024-11-07 16:36:04,127 autoreload Watching for file changes with StatReloader
INFO 2024-11-07 16:36:15,431 autoreload /workspace/cosmic-love-shop/users/views.py changed, reloading.
INFO 2024-11-07 16:36:16,308 autoreload Watching for file changes with StatReloader
ERROR 2024-11-07 16:37:35,873 log Internal Server Error: /
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/cosmic-love-shop/home/views.py", line 9, in index
    return render(request, 'home/index.html', {
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 523, in parse
    self.unclosed_block_tag(parse_until)
  File "/workspace/.pip-modules/lib/python3.12/site-packages/django/template/base.py", line 586, in unclosed_block_tag
    raise self.error(token, msg)
django.template.exceptions.TemplateSyntaxError: Unclosed tag on line 4: 'block'. Looking for one of: endblock.
WARNING 2024-11-07 16:37:36,162 log Not Found: /favicon.ico
WARNING 2024-11-07 16:38:33,514 log Not Found: /favicon.ico
